### PR TITLE
Quarkus and Flink dependencies

### PR DESCRIPTION
**Pull request by agentgonzo**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -23,6 +23,52 @@
       <url>file:${project.build.directory}/repository</url>
     </repository>
   </distributionManagement>
+
+  <properties>
+    <quarkus.version>0.19.1</quarkus.version>
```
Comment (stephenc) [inline pom.xml:28]: Are we really using this old one? we should be on 0.22.x by now

Comment (agentgonzo) [reply pom.xml:28]: That's the one we're using

Comment (agentgonzo) [reply pom.xml:28]: (at least in the slack notifier)

Comment (stephenc) [reply pom.xml:28]: Ok we can upgrade it later. Let’s merge this bad boy

**Pull request merged**